drivers/ad5686/ad5686.c

```diff
@@ -474,6 +545,9 @@ void ad5686_power_mode(struct ad5686_dev *dev,
 	uint8_t address = chip_info[dev->act_device].channel_addr[channel];
 
 	if(chip_info[dev->act_device].register_map == AD5686_REG_MAP) {
+		/* AD5674R/AD5679R have 16 channels and 2 powerdown registers */
```

Comment:
Might be worth mentioning in the header comment that "100k to GND" Power-Down mode(`AD5686_PWRM_100K`) is not available for AD5674R/AD5679R.

Reply:
Ack